app/Http/Controllers/MessageViewController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Message;
+use Illuminate\Http\Request;
+
+class MessageViewController extends Controller
+{
+    public function show($id)
+    {
+        $message = Message::find($id);
+
+        if ($message) {
+            return view('message-view', compact('message'));
+        }
+
+        abort(404, 'Message Not Found');
+    }
```

Comment:
(suggest)
Laravel のルートモデル結合を使うとスッキリします。（web.php も変更する必要があります）
https://readouble.com/laravel/8.x/ja/routing.html#route-model-binding

```suggestion
    public function show(Message $message)
    {
        return view('message-view', compact('message'));
    }
```